pkg/collector/corechecks/system/cpu_windows.go

```diff
@@ -98,8 +100,14 @@ func (c *CPUCheck) Run() error {
 		sender.Gauge("system.cpu.idle", idle*toPercent, "", nil)
```

Comment:
Would we want to use pdh here too, to get these metrics ?

Reply:
(feel free to create a card in our backlog :) )